packages/permission-controller/src/rpc-methods/getPermissions.ts

```diff
@@ -8,7 +8,7 @@ import { MethodNames } from '../utils';
 
 export const getPermissionsHandler: PermittedHandlerExport<
   GetPermissionsHooks,
-  undefined,
+  [],
   PermissionConstraint[]
 > = {
```

Comment:
Using an empty array here to replace `undefined` as the empty type resolves the compiler errors, but I'd appreciate input on whether this compromises type safety or otherwise might cause isseus down the road.